content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -38,8 +38,7 @@ If a {{< glossary_tooltip term_id="node" >}} dies, the Pods scheduled to that no
 are [scheduled for deletion](#pod-garbage-collection) after a timeout period.
 
 Pods do not, by themselves, self-heal. If a Pod is scheduled to a
-{{< glossary_tooltip text="node" term_id="node" >}} that then fails,
-or if the scheduling operation itself fails, the Pod is deleted; likewise, a Pod won't
+{{< glossary_tooltip text="node" term_id="node" >}} that then fails, the Pod is deleted; likewise, a Pod won't
```

Comment:
The wording "If a Pod is scheduled to a node that then fails" sounds vague, does it actually mean "If a running Pod exit abnormally"?

Reply:
According to the context, I **guess** it wanted to describe how the system responds when a pod exits, instead of anything related to the scheduling behavior.

Reply:
It refers to the Node failing. But it would also be the case for a Pod with restartPolicy=Never failing.

Reply:
Hi @alculquicondor .
Just checking on the status of this PR?
This PR removes the phrase, "or if the scheduling operation itself fails,". The remaining phrases are true?

Reply:
correct